### DISCUSSION TITLE
Loot tracking Q?

### DISCUSSION THREAD

**Discussion by Steiner-SE** [Lailloken/Exile-UI#252] [General]
Hi!
Just came to endgame for the first time using this script, I like to track kills and loot but have one minor issue (I might be doing something wrong).
Is there any way to collapse/hide the loot being tracked while in the instance? As is now I pick up a lot, jump out and empty then go back in. This creates a big list covering parts of my screen and hiding what is going on behind it. It would be nice if you could collaptse the list while inside the instance (I want to track my loot, not neccessarily see it onscreen at all times),
As for the killtracker I definitely am doing something wrong because every map says 0 kills. (I enter, see the tracker flash green and click it and it stops tracking).

**Comment by Lailloken** [comment]
Pressing ESC should collapse the list. If you manually type the /kills command, does the number in the chat have a thousands separator, e.g. a comma/period?

**Comment by Steiner-SE** [comment]
When I click the tracker after finishing my run I see a mention of how many kills I have in the chat window, but when I export the data to save it in Excel kills is always 0.
I'll try hitting Esc next time. Thank you.
I'll look for the formatting of the number after my next run, can't remember right now.

**Comment by Steiner-SE** [comment]
Ok, the formatting I see in the chat for my killnumber has a space in it, but no period or comma. It's "You ahve killed XX XXX monsters" basically. No kill numbers seems to be saved with my data.

**Reply by Lailloken** [reply]
I see. There was someone else with the same problem recently. It's because of specific regional settings in Windows, where thousands are separated by spaces instead of commas or periods. I have just uploaded v1.30.6 which fixes this issue.

**Comment by Steiner-SE** [comment]
oh, thank you kindly. I will grab it and test. Much appreciated.

**Comment by Steiner-SE** [comment]
Confirmed, killtracker working now. Thank you again.
